ghscrap/scrapper.py

```diff
@@ -100,7 +100,7 @@ def search_location(location, keyword=None, max_users=10, params={}):
     number_of_pages = ceil(number_of_users / 100)
     
     users = _get_users(response.json()['items'])
-    for i in range(2, number_of_pages):
+    for i in range(2, int(number_of_pages)):
```

Comment:
I would prefer you type-cast the identifier `number_of_pages` at its definition itself ( L100).